dowhy/causal_refuters/add_unobserved_common_cause.py

```diff
@@ -170,6 +178,10 @@ def refute_estimate(self):
 
         :return: CausalRefuter: An object that contains the estimated effect and a new effect and the name of the refutation used.
         """
+        if self.simulated_method_name == "Carlos":
```

Comment:
we may need to use a different method name :)  would partialR2 be a good one? what do you think is the main component of this method?